Resources/config/services.xml

```diff
@@ -90,6 +90,7 @@
             <argument type="service" id="sulu_form.repository.form" />
             <argument type="service" id="form.factory" />
             <argument type="service" id="sulu_form.checksum" />
+            <argument type="service" id="security.csrf.token_manager"/>
```

Comment:
Shouldn't we document this BC in the `UPGRADE.md` ?

Reply:
👍 added